setup.py

```diff
@@ -13,14 +14,15 @@
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
 
-P2PD_VERSION = "v0.3.16"
+P2PD_VERSION = "v0.3.17"
 
 P2PD_SOURCE_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/archive/refs/tags/{P2PD_VERSION}.tar.gz"
 P2PD_BINARY_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/releases/download/{P2PD_VERSION}/"
 
 # The value is sha256 of the binary from the release page
-EXECUTABLES = {
```

Comment:
I made `EXECUTABLES` when we also wanted to ship `p2p-keygen` binary. We now moved away from it, since `p2p-keygen` functionality can be implemented in 3 lines of Python.